lib/engines/hcktest/tests.rb

```diff
@@ -249,10 +249,22 @@ def summary_results_log
       @tests.reduce('') do |sum, test|
         extra_info = @tests_extra.dig(test['id'], 'dump') ? '(with Minidump)' : ''
         status = @tests_extra.dig(test['id'], 'status') || test['status']
-        sum + "#{status}: #{test['name']} [#{test['estimatedruntime']}] #{extra_info}\n"
+        queued_at, started_at = format_times(test)
+
+        sum + "#{status}: #{test['name']} [#{test['estimatedruntime']}] #{extra_info}#{queued_at}#{started_at}\n"
       end
     end
 
+    def format_times(test)
+      queued_at = @tests_extra.dig(test['id'], 'queued_at')&.strftime('%Y-%m-%d %H:%M:%S %Z')
```

Comment:
Probably you don't need `strftime` call at all.
`" [Queued time: #{Time.now}]"` evaluates to: `" [Queued time: 2024-05-14 12:19:27 +0900]"`